WinUIGallery/ControlPages/DesignGuidance/IconsPage.xaml

```diff
@@ -101,45 +102,67 @@
         </DataTemplate>
     </Page.Resources>
 
-    <Grid HorizontalAlignment="Stretch">
+    <Grid HorizontalAlignment="Stretch" RowSpacing="8">
         <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
             <RowDefinition Height="Auto" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
-        <StackPanel Orientation="Vertical">
-            <RichTextBlock>
-                <Paragraph>
-                    With the release of Windows 11, <Hyperlink NavigateUri="https://learn.microsoft.com/windows/apps/design/style/segoe-fluent-icons-font">Segoe Fluent Icons Font</Hyperlink>
-                    is the recommended symbol icon font.<LineBreak />
-                </Paragraph>
-            </RichTextBlock>
-            <TextBlock
-                Margin="0,8,0,0"
-                Style="{ThemeResource BodyStrongTextBlockStyle}"
-                Text="How do I get the font?" />
-            <RichTextBlock>
-                <Paragraph>
-                    On Windows 11: There's nothing you need to do, the font comes with Windows.<LineBreak />
-                    On Windows 10: The Segoe Fluent Icons is not included by default on Windows 10. You can download it <Hyperlink NavigateUri="https://learn.microsoft.com/windows/apps/design/downloads/#fonts">here</Hyperlink>
-                    .<LineBreak />
-                </Paragraph>
-            </RichTextBlock>
+        <TextBlock Text="With the release of Windows 11, Segoe Fluent Icons Font is the recommended symbol icon font."/>
```

Comment:
Minor nit: 

```suggestion
        <TextBlock Text="With the release of Windows 11, Segoe Fluent Icons is the recommended icon font."/>
```

Reply:
I copied it from the MS website. So I don't know?